runtime-decompiler/src/main/java/org/jrd/backend/decompiling/DecompilerWrapperInformation.java

```diff
@@ -49,8 +49,8 @@ public DecompilerWrapperInformation() {
     private URL decompilerDownloadURL;
     private String fileLocation;
     private String fullyQualifiedClassName;
-    private URL wrapperURL;
-    private List<URL> DependencyURLs;
+    private URLExpandable wrapperURL;
```

Comment:
I would name it ExpandableUrl, but that is suggestion only.
java naming convention for classes is CamelCase. URL is violating it.

Reply:
Makes sense, though java.net.URL also violates it in the first place.